### DISCUSSION TITLE
Stuck inside gameplay even after map has finished

### DISCUSSION THREAD

**Discussion by WasThatZero** [ppy/osu#20455] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I played a map, finished it, and waited for results but they never came. It's now been 10 minutes since it happened and I still haven't gotten to the results.
Beatmap: https://osu.ppy.sh/beatmapsets/955866#osu/2003881

### Screenshots or videos

![image](https://user-images.githubusercontent.com/105557160/192101543-03b09310-c46c-414a-b91e-2bc1c2ed1273.png)
![image](https://user-images.githubusercontent.com/105557160/192101546-0d597a9b-7946-4fcb-8a07-c86c04636be4.png)
![image](https://user-images.githubusercontent.com/105557160/192101549-4b1ec8f4-e0d5-46a2-b1fc-a47a9d6f419b.png)
![image](https://user-images.githubusercontent.com/105557160/192101551-a6de17e9-89da-4e70-8283-532b049a7fa7.png)

### Version

2022.923.1

### Logs

Logs: https://drive.google.com/drive/folders/1ZeVeUfKY2fenTRyHkwWqvBgpB8nnP3o4?usp=sharing

**Comment by frenzibyte** [comment]
<img width="435" alt="CleanShot 2022-09-24 at 16 56 34@2x" src="https://user-images.githubusercontent.com/22781491/192101852-2e5d69f5-25a0-4c36-86d9-b744df26535d.png">

Please double-check again. Have you changed your game folder location? Also please attach the beatmap.

**Reply by WasThatZero** [reply]
Thanks! Updated the issue post now, including beatmap and link to logs.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/20548

**Reply by ghost** [reply]
not sure if this was intended but the discussion linked in your answer just links back here

maybe you meant to link #20548?
